src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliMiddleware.cs

```diff
@@ -23,23 +23,24 @@ internal static class AngularCliMiddleware
 
         public static void Attach(
             ISpaBuilder spaBuilder,
-            string npmScriptName)
+            string scriptName)
```

Comment:
Same as @Eilon said above, even though your new name is more accurate it's still a breaking change. It might make sense to file an issue just for renaming parameters like this so we can take it or not without blocking this.

Reply:
I reverted this change

Reply:
You reverted a bunch of other breaking changes, but it seems this and one other (ReactDevelopmentServerMiddleware.Attach) slipped through.

Reply:
I assumed changes to internal method signatures did not count as breaking changes, is this not the case?

Reply:
Oh, you're totally right. I missed that part in both cases. You're good to go then.